### ISSUE TITLE
When can we start writing the documentation

### ISSUE THREAD

**Issue by jeff-h** [MacGapProject/documentation#1]
> When can we start writing the documentation? feature freeze?

I think we should actually start immediately, and have already been working on the foundations of it in other issues. Most of the MG2 API is unlikely to change much if at all once it's been coded.

I guess before anyone can really help out, I need to get the infrastructure in place. Next steps:
- me: move gh-pages from /MacGap2 to /documentation
- me: sort out CNAME pointer for docs.macgap.com
- migrate across all existing document ion and go through the new code to modify the docs as needed

**Comment by rawcreative** [comment]
You can start writing documentation whenever, unless bugs are found and something NEEDS to change, or there’s requests for something to change, everything that’s currently implemented will stay the way it is. I’ll just be adding new commands or extending/adding onto the current commands from here on out. I don’t really have anything planned that would significantly alter the way things are documented or how everything in general works. I have debated on writing a dispatch system and a backend interface for interacting with MacGap via js without the need for the webview/window. It’d mainly be for handling things when multiple windows are used or for windowless apps. I’d probably make it work independently from the main stuff so it would optional.. haven’t really flushed out the details yet.

**Comment by jeff-h** [comment]
Documentation is now at http://macgapproject.github.io/documentation

**Comment by jeff-h** [comment]
CNAME is sorted. Docs are now available at http://docs.macgap.com

**Comment by fluke** [comment]
Would be nice to have some instructions on how to use it. Does it work the same way as the MacGap1 project? Using gem install?

**Comment by jeff-h** [comment]
I agree re instructions — it's a gaping hole in our docs at present. In most ways it is roughly the same as MacGap 1, at least from a JavaScript developer's point of view (note that it provides a different API though).

http://docs.macgap.com/doc/getting-started.html
http://docs.macgap.com/doc/app-configuration.html

both need work, if anyone has some time to work on them.

Re: gem for MG2, see https://github.com/MacGapProject/MacGap2/issues/31
